visualization/app/codeCharta/ui/customConfigs/customConfigs.component.html

```diff
@@ -1,16 +1,28 @@
 <md-menu class="custom-configs-component">
-	<md-button
-		class="custom-configs-button"
-		ng-click="$ctrl.loadCustomConfigs(); $mdMenu.open($event); "
-		title="Open your saved custom configs"
-	>
+	<md-button class="custom-configs-button" ng-click="$ctrl.initView(); $mdMenu.open($event);" title="Open your saved custom configs">
 		<i class="fa fa-eye"></i>
 	</md-button>
 
 	<md-menu-content class="custom-configs-drop-down" width="7">
 		<md-menu-item class="custom-configs-item">
-			<md-button class="custom-configs-dialog-name" ng-disabled="true">Custom Configs applicable to current map and mode</md-button>
-			<md-button ng-click="$ctrl.showAddCustomConfigSettings()" class="action-button add" title="Create new custom config">
```

Comment:
In line 8 and 82 it is "**C**ustom **C**onfig". In line 25 it is small again. I don't know, if we want to write it capitalized or not, but we probably should be consistent

Reply:
Fixed